src/linearmixedmodel.jl

```diff
@@ -314,7 +315,8 @@ function createAL(allterms::Vector{Union{AbstractReMat{T},FeMat{T}}}) where {T}
             cj = allterms[j]
             if !isnested(cj, ci)
                 for l = i:k
-                    L[Block(l, i)] = Matrix(getblock(L, l, i))
+                    ind = packedlowertri(l, i)
+                    L[ind] = Matrix(L[ind])
```

Comment:
Should we add a `getblock` method or something similar that just wraps the use of `packedlowertri`?

Reply:
That probably would have been the simplest approach conceptually but I was in the middle of changing all those `getblock` calls to indexing with `packedlowertri` before it occurred to me.  Maybe it would be best to wait to see if and how the Xy amalgamation works out before deciding on a `getblock` strategy.